### ISSUE TITLE
Bug Report for gradient-descent

### ISSUE THREAD

**Issue by oojBuffalo** [neetcode-gh/leetcode#5769]
Bug Report for https://neetcode.io/problems/gradient-descent

Please describe the bug below and include any steps to reproduce the bug or screenshots if possible.

---

Albeit a very minor detail, the problem description and function signature do not appear to be consistent with Test Case 1.

For reference, Test Case 1 (same as Example 1):

> 
```
Input:
iterations = 0, learning_rate = 0.01, init = 5

Output:
5
```

The problem description states: 

> Use Python's round() function to round the output to 5 decimal places.

The function signature is:

> `def get_minimizer(self, iterations: int, learning_rate: float, init: int) -> float`

Granted there are no iterations occurring, and the initial value ought to remain unchanged, it seems like bad practice to have the function return an integer value, when the problem description and type hints indicate a float should be returned.

<img width="1034" height="545" alt="Image" src="https://github.com/user-attachments/assets/a252f767-a3dc-47c0-a1c3-d781e9cee6ee" />
